matryoshka_tester/helpers.py

```diff
@@ -117,25 +117,56 @@ def get_selected_runtime() -> OciRuntimeBase:
 
 
 @dataclass(frozen=True)
-class ContainerBuild(ToParamMixin):
+class GitRepositoryBuild(ToParamMixin):
     """Test information storage for running docker builds from a Dockerfile in
-    the dockerfiles/ subfolder. This fixture is to be used in conjunction with
+    the dockerfiles/ subfolder. This class is to be used in conjunction with
```

Comment:
The documentation for that class is now incorrect.

Reply:
I've improved the documentation for this class and for the fixtures as well